WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -96,6 +96,14 @@ public class BlockEditorScreen: ScreenObject {
         $0.staticTexts["You have unsaved changes."]
     }
 
+    private let turnOnImageOptimizationGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Yes, leave on"]
+    }
+
+    private let turnOffImageOptimizationGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["No, turn off"]
+    }
```

Comment:
For getters, we follow the naming pattern where the element type is included:

```suggestion
    private let turnOnImageOptimizationButtonGetter: (XCUIApplication) -> XCUIElement = {
        $0.buttons["Yes, leave on"]
    }

    private let turnOffImageOptimizationButtonGetter: (XCUIApplication) -> XCUIElement = {
        $0.buttons["No, turn off"]
    }
```

Reply:
Thanks @pachlava for the suggestion, I've applied this in https://github.com/wordpress-mobile/WordPress-iOS/pull/22095/commits/88e54245ae1d1eafc80e92ce4ec51a4ae2820140.